projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/pojo/TableObject.java

```diff
@@ -0,0 +1,48 @@
+package org.batfish.datamodel.pojo;
+
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
+
+/**
+ * {@link TableObject TableObject} is an Object representation of an bgpTable/routingTable file.
+ * {@link #_name} stores the name of the file, and {@link #_content} contains the file content.
+ */
+public class TableObject {
```

Comment:
What about this object is specific to a Table vs any other kind of file?

Reply:
It should be able to represent any actual files. I changed the name to FileObject~